README.md

```diff
@@ -164,4 +192,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+This is a level 2 language because its context-free and can be passed down a finite state machine.  
```

Comment:
# 0

It's a type-3 language because it is regular, and can be parsed using a state machine.